WalletWasabi/Helpers/RuntimeParams.cs

```diff
@@ -15,10 +15,13 @@ public class RuntimeParams
 	private RuntimeParams()
 	{
 	}
-
+	
```

Comment:
I use VS but since I began to contribute to WW I have ton of similar issues with it... No auto-formatting, no warnings/erros highlighting, Ctrl+k not working etc...
I'll try to reinstall